### PR TITLE
VS Code extension docs: Changes to database downloads 

### PR DESCRIPTION
**Pull request by shati-patel**
After https://github.com/github/vscode-codeql/pull/3047 gets released, we'll no longer add database source archives to the workspace by default (since this caused bugs in certain workspaces).

The main feature that's affected is the "[AST viewer](https://codeql.github.com/docs/codeql-for-visual-studio-code/exploring-the-structure-of-your-source-code/)", so I've added an extra step to the instructions there. 

I've also described a new setting in "Customizing settings", in case users want to restore the old behavior:

image

### CODE REVIEW

File: docs/codeql/codeql-for-visual-studio-code/customizing-settings.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ Changing the labels of query history items
 
 The query history **Format** setting controls how the extension lists queries in the query history. By default, each item has a label with the following format::
     
-    %q on %d - %s, %r result count [%t]
+    %q on %d - %s %r [%t]
```
Comment (shati-patel) [inline docs/codeql/codeql-for-visual-studio-code/customizing-settings.rst:42]: Unrelated change: I just spotted that this was out of date.

File: docs/codeql/codeql-for-visual-studio-code/exploring-the-structure-of-your-source-code.rst [hunk 2]
```diff
@@ -18,20 +18,26 @@ CodeQL for Visual Studio Code contains an AST viewer. The viewer consists of a g
 Viewing the abstract syntax tree of a source file
 --------------------------------------------------
 
-1. Open a source file from a CodeQL database. For example, you can navigate to a source file in the File Explorer.
+1. Open the CodeQL Databases view and right-click the database that you want to explore. Click **Add Database Source to Workspace**.
+
+   .. image:: ../images/codeql-for-visual-studio-code/add-database-source-to-workspace.png
```
Comment (shati-patel) [inline docs/codeql/codeql-for-visual-studio-code/exploring-the-structure-of-your-source-code.rst:23]: I've stuck with a dark mode screenshot, for consistency with the rest of this doc.

**Review by charisk — APPROVED** [review]
LGTM

**Review by jf205 — APPROVED** [review]

**Review by subatoi — APPROVED** [review]
Thank you, looks great!
